libcxx/modules/std/memory.inc

```diff
@@ -156,7 +156,9 @@ export namespace std {
   using std::reinterpret_pointer_cast;
   using std::static_pointer_cast;
 
+#ifndef _LIBCPP_HAS_NO_RTTI
   using std::get_deleter;
+#endif // _LIBCPP_HAS_NO_RTTI
```

Comment:
This tested here, but should land separately. (Currently the CI does not test modules so it passes.)